src/components/CreatePartyPage/Form/Web_shop/ShopItem/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react"
+
+class Shop_item extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      selected: false,
+      infoToggled: false
+    }
+    this.toggleSelected = this.toggleSelected.bind(this)
+    this.toggleInfo = this.toggleInfo.bind(this)
+  }
+
+  toggleSelected() {
```

Comment:
Ta bort bind(this) på båda dina funktioner och gör om dem till arrow funktions istället, kör man ES6 arrow functions så slipper vi bind(this) och liknande. 

Se nedanstående:

toggleSelected =() => {
    this.setState({ selected: !this.state.selected })
  }

  toggleInfo = () => {
    this.setState({ infoToggled: !this.state.infoToggled })
  }